packages/vat-data/package.json

```diff
@@ -24,7 +24,9 @@
     "@agoric/internal": "^0.3.2",
     "@agoric/store": "^0.9.2",
     "@agoric/swingset-liveslots": "^0.10.2",
-    "@agoric/vow": "^0.1.0"
+    "@agoric/vow": "^0.1.0",
+    "@endo/exo": "^1.1.0",
+    "@endo/patterns": "^1.1.0"
```

Comment:
Double check this. Seems stale

Reply:
Was indeed stale. Now that it is updated, the `yarn.lock` changes also disappeared. Done.